lib/routes/token.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var validators = require('./validators')
+var HEX_STRING = validators.HEX_STRING
+
+module.exports = function (log, isA, error, db) {
+
+  var routes = [
+    {
+      method: 'POST',
+      path: '/token/verify',
+      config: {
+        validate: {
+          payload: {
+            code: isA.string().min(32).max(32).regex(HEX_STRING).required()
+          }
+        }
+      },
+      handler: function (request, reply) {
+        log.begin('Token.verify', request)
+        // TODO - Do something real here
+        reply({})
+      }
+    },
+    {
+      method: 'GET',
+      path: '/token/status',
```

Comment:
As an aside, the spec mentions `/token/:id/status`. Both recoveryEmailStatus and passwordForgotStatus use [hawk credentials to pass the tokens](https://github.com/mozilla/fxa-js-client/blob/7ec11efc73bfde196e37ea5e5454428c2ec72c9f/client/FxAccountClient.js#L521). [`/account/status` passes the uid as a GET param](https://github.com/mozilla/fxa-js-client/blob/7ec11efc73bfde196e37ea5e5454428c2ec72c9f/client/FxAccountClient.js#L672).

I'm unsure of why one is passed using Hawk creds, and the other a GET parameter.